flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/optimize/program/FlinkChangelogModeInferenceProgram.scala

```diff
@@ -470,20 +470,21 @@ class FlinkChangelogModeInferenceProgram extends FlinkOptimizeProgram[StreamOpti
 
       case _: StreamPhysicalGroupAggregate | _: StreamPhysicalGroupTableAggregate |
            _: StreamPhysicalLimit | _: StreamPhysicalPythonGroupAggregate |
-           _: StreamPhysicalPythonGroupTableAggregate =>
-        // Aggregate, TableAggregate and Limit requires update_before if there are updates
+           _: StreamPhysicalPythonGroupTableAggregate |
```

Comment:
nit: use `StreamPhysicalGroupWindowAggregateBase` instead of three sub-classes

Reply:
I use three sub-classes as previous version. I thought author maybe have reason to use three sub-classes instead of `StreamPhysicalGroupWindowAggregateBase `, for example, prevent the misuse of the parent class when adding a new child class in the future ?